code/modules/mob/living/simple_animal/abnormality/waw/snow_whites_apple.dm

```diff
@@ -0,0 +1,264 @@
+/mob/living/simple_animal/hostile/abnormality/F0442
```

Comment:
Best to use abnormality name here, not the number designation to make things easier on admins

Reply:
Honestly i initially did that as a joke. But ill change it now.